.github/workflows/ci.yml

```diff
@@ -126,6 +126,12 @@ jobs:
         with:
           version: 'v1.16.1' # Manually pinned
           args: 'release -f .goreleaser.docker.yaml --clean'
+      - name: 'Apply artifact registy cleanup policy'
+        run: |-
+          gcloud artifacts repositories set-cleanup-policies ci-images\
+            --project=${{ env.INTEG_TEST_PROJECT_ID }}\
+            --location=us --policy=internal/testhelper/ci-ar-cleanup-policy.json\
+            --dry-run
```

Comment:
If we think this is a good idea, I will change this flag to --no-dry-run to apply the policy.

Reply:
Do we need to run it for each ci/cd run? I'm wondering should we have another scheduled workflow to set the policy instead of using ci.yml?

Reply:
I thought about it, I think making another workflow seems to be a better way. Because it can other prevent people changing the config file and run CI on that branch, which would lead to changing the cleanup policy before the branch got merged.

Reply:
I will update the code.

Reply:
This shouldn't be set repeatedly in CI.